docs/DeveloperGuide.md

```diff
@@ -313,36 +313,45 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 
-1.  Pseudo steps
-2.  User requests to delete a specific assignment in the list.
-3.  Academic Schedule Manager deletes the assignment.
+1.  User retrieves NUSMods timetable URL from the NUSMods website.
+2.  User requests to import NUSMods timetable using their NUSMods timetable URL.
+3.  Academic Schedule Manager retrieves lesson data from NUSMods API.
+4.  Academic Schedule Manager adds all the lessons according to the data retrieved.
+5.  Academic Schedule Manager shows a success message with details of the lessons added.
 
     Use case ends.
 
 **Extensions**
 
-* 3a. The given index is invalid (index is referring to a lesson or index is out of range).
+* 2a. The given URL is invalid (not a valid NUSMods timetable URL).
 
-    * 3a1. Academic Schedule Manager shows an error message.
+    * 2a1. Academic Schedule Manager shows an error message.
 
       Use case resumes at step 2.
 
+* 2b. User already has a timetable imported before.
+
+    * 2b1. Academic Schedule Manager deletes all lessons from previously imported timetable.
+    
+      Use case resumes at step 3.
```

Comment:
Similar to the above comment, maybe it would be better to change the phrasing to describe how the ASM **displays** the lessons from the newly imported timetable

Reply:
Edited accordingly. Thank you.